Ex1/week6_4.py

```diff
@@ -0,0 +1,23 @@
+from PIL import Image
+
+
+def decoding_message_in_image(path):
+    """
+    In each column of the image there is a pixel painted black.
```

Comment:
Describe in the documentation what it does, like - The function decodes a hidden message hidden in bla bla pixels and bla bla order